src/question_rating.ts

```diff
@@ -173,6 +173,15 @@ export class QuestionRatingModel extends Question {
       });
   }
 
+  @property({ defaultValue: false }) inputHasValue: boolean;
+
+  public get showSelectedItemLocText(): boolean {
+    return !this.readOnly && !this.inputHasValue && !!this.selectedItemLocText;
+  }
+  public get selectedItemLocText(): LocalizableString {
```

Comment:
Can it return empty array?

Reply:
`this.visibleRateValues.filter(v => v.value == this.value)[0]?.locText`
this code is safe, zero element of empty array returns undefined, `?.` protects of reading `locText` for undefined object